### ISSUE TITLE
Android build failed with an exception

### ISSUE THREAD

**Issue by marcio-pessoa** [fluttercommunity/wakelock_plus#3]
Hi dears.

I got an issue during Android build:

```log
...
Launching lib/main.dart on Pixel 7 in debug mode...

FAILURE: Build failed with an exception.

* Where:
Build file '~/.pub-cache/hosted/pub.dev/wakelock_plus-1.1.0/android/build.gradle' line: 30

* What went wrong:
A problem occurred evaluating project ':wakelock_plus'.
> Could not find method namespace() for arguments [dev.fluttercommunity.plus.wakelock] on extension 'android' of type com.android.build.gradle.LibraryExtension.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org
BUILD FAILED in 1s
Exception: Gradle task assembleDebug failed with exit code 1
Exited (sigterm)
```

Could you help me please?

**Comment by diegotori** [comment]
@marcio-pessoa looks like you're using an older version of the Android Gradle Plugin.

There is a way that I can conditionally add `namespace` if the property exists. I'll create a fix for this very shortly.

In the meantime, if you're able to, you may want to upgrade your app's Android Gradle Plugin to at least `7.3.x` or higher.

**Comment by diegotori** [comment]
@marcio-pessoa addressed in `wakelock_plus`, version `1.1.1`.

**Comment by marcio-pessoa** [comment]
Thank you very much dear @diegotori .
The issue was solved with your help.
